README.md

```diff
@@ -287,7 +287,7 @@ Available vehicle remote interface implementations are:
 - `bmw`: BMW (i3)
 - `carwings`: Nissan (Leaf pre 2019)
 - `citroen`, `ds`, `opel`, `peugeot`: All PSA brands
-- `fiat`: Fiat (500e)
+- `fiat`: Fiat (Neuer 500)
```

Comment:
Ist das kein 500e? Was heisst "Neuer"?

Reply:
Es gab einen alten Fiat 500 (auch als Elektro Version). Das aktuelle und von mir genutzte Modell heißt bei Fiat offiziell "Neuer 500". Ob das gleiche auch bei anderen/älteren Modellen in dieser Art funktioniert, kann ich nicht sagen.
Um Verwirrungen zu vermeiden hatte ich dies angepasst.
Fiat's "Neuer 500" gibt es ausschließlich als Elektro (Abkehr vom Verbrenner für den 500)

Reply:
Und für den alten 500e geht das API nicht?

Reply:
ping @int5749

Reply:
Oups, da war ein weiter Comment. Thanks for the Ping.

Das kann ich nicht sagen, da ich diese API nicht kenne. Daher der Vorschlag dies so zu benennen um Klarheit zu haben und man könnte (wie bei anderen) andere Modelle erweitern. Oder wir starten eine Umfrage in Diskussionen? Da war ja min. noch ein weiterer Nutzer für Fiat (aber da kenne ich das Modell nicht)

Reply:
Gerne. Oder wir lassen einfach 500e und ergänzen Neuer 500 weil bisher niemand gesagt hat, dass 500e falsch ist ;)

Reply:
> 
> 
> Oder wir lassen einfach 500e und ergänzen Neuer 500 weil bisher niemand gesagt hat, dass 500e falsch ist ;)
Sounds like a plan und ist der geringere Aufwand anstatt zu warten bis x Personen zurückgemeldet haben und gerade denkt man(n) man ist fertig, kommt noch eine Nachmeldung. Ich passe dies an.

Reply:
@andig 
Updated